src/Trixi.jl

```diff
@@ -248,7 +249,7 @@ export ControllerThreeLevel, ControllerThreeLevelCombined,
        IndicatorNeuralNetwork, NeuralNetworkPerssonPeraire, NeuralNetworkRayHesthaven,
        NeuralNetworkCNN
 
-export PositivityPreservingLimiterZhangShu
+export PositivityPreservingLimiterZhangShu, SubcellLimiterIDPCorrection
```

Comment:
Yes, good idea. Including all my upcoming PRs, the export code would look like
```
export DG,
       DGSEM, LobattoLegendreBasis,
       FDSBP,
       VolumeIntegralWeakForm, VolumeIntegralStrongForm,
       VolumeIntegralFluxDifferencing,
       VolumeIntegralPureLGLFiniteVolume,
       VolumeIntegralShockCapturingHG, IndicatorHennemannGassner,
       VolumeIntegralSubcellLimiting, SubcellLimiterIDP, SubcellLimiterMCL,
       SubcellLimiterIDPCorrection, BoundsCheckCallback,
       VolumeIntegralUpwind,
       SurfaceIntegralWeakForm, SurfaceIntegralStrongForm,
       SurfaceIntegralUpwind,
       MortarL2
```

Probably, it would be the best to move my new stuff to its own "export section". Although, the `VolumeIntegralSubcellLimiting` fits in well here. What do you think?

Reply:
Yes, that sounds like a good idea. The implementation is already in separate files, so moving the `export` statements into a separate block sounds good :+1:

Reply:
Please update this as discussed and ping me again for a review. Then, we should be able to merge this when CI is green. Thanks a lot!